src/Widgets/EligibilityModal/components/Cards/index.tsx

```diff
@@ -0,0 +1,27 @@
+import React, {FC} from 'react'
+import { Card } from 'types'
+import AmexCard from 'assets/cards/amex'
+import VisaCard from 'assets/cards/visa'
+import MasterCard from 'assets/cards/mastercard'
+import CbCard from 'assets/cards/cb'
+import s from './Cards.module.css'
+type Props = {
+  cards: Card[]
+}
+
+const Cards: FC<Props> = ({ cards }) => {
+  return (
+    <div className={s.cardContainer}>
```

Comment:
Also I think it'd be nice if we add Static CSS now.
I think a static class for cardContainer would be enough